code/modules/jobs/job_types/clown.dm

```diff
@@ -2,7 +2,7 @@
 	title = JOB_CLOWN
 	description = "Entertain the crew, make bad jokes, go on a holy quest to find bananium, HONK!"
 	department_head = list(JOB_HEAD_OF_PERSONNEL)
-	faction = FACTION_STATION
+	faction = list(FACTION_STATION,"clowns")
```

Comment:
create a clown faction define

Reply:
The issue with that is where to place it. There is no /code/_DEFINES/factions.dm. For instance the faction defines for antags is in

https://github.com/tgstation/tgstation/blob/5b6fa14a9f0d9c9c8b783a4453d602420482e9c2/code/__DEFINES/antagonists.dm

Reply:
Make one then?

Reply:
This is babies first PR. I mean I could but then i'd be changing even more stuff and possibly screwing stuff up. The heretic faction define lives in the antagonists.dm, right next to the other heretic defines, same for other faction defines which makes sense.

Reply:
make one please

Reply:
Its done